webapp/frontend/lib/common/calculations.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:get_10101/common/model.dart';
+import 'package:get_10101/services/quote_service.dart';
+
+Amount calculateFee(Usd? quantity, BestQuote? quote, bool isLong) {
```

Comment:
hmm.. I guess it's reasonable, you don't want to make a backend call for that, but yet we didn't want this kind of logic in dart.

I guess there is no way around that. 🤔

Reply:
Unfortunately not. I think this is better then doing a roundtrip to the backend for every re-calculation. This could get quite expensive.